src/manager/mod.rs

```diff
@@ -8,5 +8,5 @@ pub use crate::lotus::message::ipc::SubnetInfo;
 
 pub mod evm;
```

Comment:
Are we going to support `evm` in the future? I was thinking `fevm` makes use of `evm`, so that in the future, if we need `evm` only, then we dont need much work. Also the current code structure is better organized.

Reply:
I see. Let's leave it for now, we can refactor in the future.